src/js/routes/jobs.js

```diff
@@ -37,6 +38,17 @@ const jobsRoutes = [
           }
         ]
       },
+      {
+        type: Route,
+        component: JobsOverview,
+        path: "overview2",
```

Comment:
Let's not introduce a second route here, but let's only change it for development and testing until we think it is completely ready.